### PR TITLE
Fix failing func test for /account API - Closes #3239

### LINKED ISSUE #3239: Functional Get account is failing randomly
### Expected behavior
functional test GET /accounts is failing randomly

### Actual behavior
```
[2019-04-01T14:59:13.467Z]   1) GET /accounts

[2019-04-01T14:59:13.467Z]        ?

[2019-04-01T14:59:13.467Z]          sort

[2019-04-01T14:59:13.467Z]            using sort = balance:desc should return accounts in descending order by balance:

[2019-04-01T14:59:13.467Z] 

[2019-04-01T14:59:13.467Z]       AssertionError: expected [ Array(10) ] to deeply equal [ Array(10) ]

[2019-04-01T14:59:13.467Z]       + expected - actual

[2019-04-01T14:59:13.467Z] 

[2019-04-01T14:59:13.467Z]            "secondPublicKey": ""

[2019-04-01T14:59:13.467Z]            "unconfirmedBalance": "7990000000"

[2019-04-01T14:59:13.467Z]          }

[2019-04-01T14:59:13.467Z]          {

[2019-04-01T14:59:13.467Z]       -    "address": "13397677894327980967L"

[2019-04-01T14:59:13.467Z]       -    "balance": "600000000"

[2019-04-01T14:59:13.467Z]       -    "delegate": {

[2019-04-01T14:59:13.467Z]       -      "approval": 0

[2019-04-01T14:59:13.467Z]       -      "missedBlocks": 0

[2019-04-01T14:59:13.467Z]       -      "producedBlocks": 0

[2019-04-01T14:59:13.467Z]       -      "productivity": 0

[2019-04-01T14:59:13.467Z]       -      "rank": [null]

[2019-04-01T14:59:13.467Z]       -      "rewards": "0"

[2019-04-01T14:59:13.467Z]       -      "username": "udgopklplc"

[2019-04-01T14:59:13.467Z]       -      "vote": "0"

[2019-04-01T14:59:13.467Z]       -    }

[2019-04-01T14:59:13.467Z]       -    "publicKey": "84fa44aa3b3e1d3fcf57158795cc69401847e6418010e18b36e5a2861ec17acf"

[2019-04-01T14:59:13.467Z]       -    "secondPublicKey": ""

[2019-04-01T14:59:13.467Z]       -    "unconfirmedBalance": "600000000"

[2019-04-01T14:59:13.467Z]       -  }

[2019-04-01T14:59:13.467Z]       -  {

[2019-04-01T14:59:13.467Z]            "address": "17971698090269522089L"

[2019-04-01T14:59:13.467Z]            "balance": "3900000000"

[2019-04-01T14:59:13.467Z]            "publicKey": "58509825a745f125c7c39fb5edc69eff2b01be85bf1f707196eb98562126b943"

[2019-04-01T14:59:13.467Z]            "secondPublicKey": ""

[2019-04-01T14:59:13.467Z] --

[2019-04-01T14:59:13.467Z]            "publicKey": ""

[2019-04-01T14:59:13.467Z]            "secondPublicKey": ""

[2019-04-01T14:59:13.467Z]            "unconfirmedBalance": "1500000000"

[2019-04-01T14:59:13.467Z]          }

[2019-04-01T14:59:13.467Z]       +  {

[2019-04-01T14:59:13.467Z]       +    "address": "13397677894327980967L"

[2019-04-01T14:59:13.467Z]       +    "balance": "600000000"

[2019-04-01T14:59:13.467Z]       +    "delegate": {

[2019-04-01T14:59:13.467Z]       +      "approval": 0

[2019-04-01T14:59:13.467Z]       +      "missedBlocks": 0

[2019-04-01T14:59:13.467Z]       +      "producedBlocks": 0

[2019-04-01T14:59:13.467Z]       +      "productivity": 0

[2019-04-01T14:59:13.467Z]       +      "rank": [null]

[2019-04-01T14:59:13.467Z]       +      "rewards": "0"

[2019-04-01T14:59:13.467Z]       +      "username": "udgopklplc"

[2019-04-01T14:59:13.467Z]       +      "vote": "0"

[2019-04-01T14:59:13.467Z]       +    }

[2019-04-01T14:59:13.467Z]       +    "publicKey": "84fa44aa3b3e1d3fcf57158795cc69401847e6418010e18b36e5a2861ec17acf"

[2019-04-01T14:59:13.467Z]       +    "secondPublicKey": ""

[2019-04-01T14:59:13.467Z]       +    "unconfirmedBalance": "600000000"

[2019-04-01T14:59:13.467Z]       +  }

[2019-04-01T14:59:13.467Z]        ]

[2019-04-01T14:59:13.467Z]       

[2019-04-01T14:59:13.467Z]       at accountsEndpoint.makeRequest.then.res (framework/test/mocha/functional/http/get/accounts/accounts.js:330:15)

[2019-04-01T14:59:13.467Z]       at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)

[2019-04-01T14:59:13.467Z]       at Promise._settlePromiseFromHandler (node_modules/bluebird/js/release/promise.js:512:31)

[2019-04-01T14:59:13.467Z]       at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:569:18)

[2019-04-01T14:59:13.467Z]       at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:614:10)

[2019-04-01T14:59:13.467Z]       at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:694:18)

[2019-04-01T14:59:13.467Z]       at _drainQueueStep (node_modules/bluebird/js/release/async.js:138:12)

[2019-04-01T14:59:13.467Z]       at _drainQueue (node_modules/bluebird/js/release/async.js:131:9)

[2019-04-01T14:59:13.467Z]       at Async._drainQueues (node_modules/bluebird/js/release/async.js:147:5)

[2019-04-01T14:59:13.467Z]       at Immediate.Async.drainQueues [as _onImmediate] (node_modules/bluebird/js/release/async.js:17:14)
```

### Which version(s) does this affect? (Environment, OS, etc...)
1.6.0-alpha.4

### PR DESCRIPTION
**Pull request by michielmulders**
### What was the problem?
Failing tests for sorting /account API endpoint results.

### How did I fix it?
Convert balance from `string` to `int` for correct sorting.

### How to test it?
Run functional tests.

### Review checklist

* The PR resolves #3239 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: framework/test/mocha/functional/http/get/accounts/accounts.js [hunk 1]
```diff
@@ -302,6 +302,9 @@ describe('GET /accounts', () => {
 
 			it('using no sort return accounts sorted by both balance and address in asending order as default behavior', async () => {
 				return accountsEndpoint.makeRequest({}, 200).then(res => {
+					_.each(res.body.data, item => {
+						item.balance = parseInt(item.balance);
```
Comment (pablitovicente) [inline framework/test/mocha/functional/http/get/accounts/accounts.js]: shouldn't this be bignum?

Comment (michielmulders) [reply framework/test/mocha/functional/http/get/accounts/accounts.js]: Yes, just pushed it!

File: framework/test/mocha/functional/http/get/accounts/accounts.js [hunk 2]
```diff
@@ -302,6 +303,9 @@ describe('GET /accounts', () => {
 
 			it('using no sort return accounts sorted by both balance and address in asending order as default behavior', async () => {
 				return accountsEndpoint.makeRequest({}, 200).then(res => {
+					_.each(res.body.data, item => {
```
Comment (pablitovicente) [inline framework/test/mocha/functional/http/get/accounts/accounts.js]: Don't you need to assign this? I don't see any reference in the Lodash documentation about `each` mutating the original array

Comment (michielmulders) [reply framework/test/mocha/functional/http/get/accounts/accounts.js]: Yes it does!

Comment (pablitovicente) [reply framework/test/mocha/functional/http/get/accounts/accounts.js]: ohh ok and res.body.data is an object!

Comment (michielmulders) [reply framework/test/mocha/functional/http/get/accounts/accounts.js]: Check next commit, implemented safer sorting without `_.each`

File: framework/test/mocha/functional/http/get/accounts/accounts.js [hunk 3]
```diff
@@ -304,7 +305,10 @@ describe('GET /accounts', () => {
 				return accountsEndpoint.makeRequest({}, 200).then(res => {
 					const balances = _.clone(res.body.data);
 					expect(
-						_.orderBy(balances, ['balance', 'address'], ['asc', 'asc'])
+						balances.sort((a, b) => {
+							const aBignumBalance = new Bignum(a.balance);
+							return aBignumBalance.minus(b.balance) || a.address - b.address;
```
Comment (shuse2) [inline framework/test/mocha/functional/http/get/accounts/accounts.js]:
```suggestion
							return aBignumBalance.minus(b.balance) || (a.address > b.address) ? 1 : -1;
```

Comment (shuse2) [reply framework/test/mocha/functional/http/get/accounts/accounts.js]: above the comparison might be the opposite for address

Comment (michielmulders) [reply framework/test/mocha/functional/http/get/accounts/accounts.js]: I used `localeCompare` to be safe in the end.

**Pull request merged**